.github/workflows/update-syscalls.yml

```diff
@@ -0,0 +1,44 @@
+name: Update Syscalls
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '* * * * 2'
```

Comment:
This should be monthly job, not weekly, eg `0 2 1 * *` (At 02:00 on day-of-month 1)